src/components/Home.js

```diff
@@ -1,25 +1,15 @@
+/* eslint-disable no-console */
 /* eslint-disable no-unused-vars */
 /* eslint-disable implicit-arrow-linebreak */
```

Comment:
- [ ] Also kindly try to fix the linter errors instead of disabling them, you have disabled many